crawler/Cargo.toml

```diff
@@ -0,0 +1,60 @@
+[package]
+name = "snarkos-crawler"
+version = "2.0.0"
+authors = [ "The Aleo Team <hello@aleo.org>" ]
+description = "A decentralized operating system"
+homepage = "https://aleo.org"
+repository = "https://github.com/AleoHQ/snarkOS"
+keywords = [
+    "aleo",
+    "cryptography",
+    "blockchain",
+    "decentralized",
+    "zero-knowledge"
+]
+categories = [ "cryptography", "operating-systems" ]
+license = "GPL-3.0"
+edition = "2018"
+
+[dependencies.snarkos]
```

Comment:
note: the crawler would be more lightweight and faster to compile if the network components of snarkOS were moved to a crate of their own, as proposed in the past (it would be outside of this PR's scope, though)

Reply:
Agreed!